i18n/sample/src/Profiles.i18n.json

```diff
@@ -18,7 +18,7 @@
     "promptcredentials.option.prompt.username.placeholder": "User Name",
     "promptcredentials.option.prompt.username": "Enter the user name for the connection",
     "promptcredentials.invalidusername": "Please enter your z/OS username. Operation Cancelled",
-    "promptcredentials.option.prompt.password.placeholder": "Password",
+    "promptcredentials.option.prompt.passWord.placeholder": "Password",
```

Comment:
I believe it was decided to go with lowercase `password`. not sure how this got in here.